src/python/pants/build_graph/address.py

```diff
@@ -256,6 +252,9 @@ def __ne__(self, other):
     return not self == other
 
   def __repr__(self) -> str:
+    return f"Address({self.spec_path}, {self.target_name})"
```

Comment:
This is new. Meant to make it easier to debug. We use `str()` instead of `repr()` almost everywhere, so this shouldn't impact any pre-existing code.

Reply:
This is also more correct according to the contract of `__repr__`!